code/src/com/sixsq/nuvla/server/resources/nuvlabox/status_utils.clj

```diff
@@ -67,3 +82,212 @@
     (when-let [resource-id (nb-utils/get-service "swarm" infrastructure-service-group)]
       (let [response (db/scripted-edit resource-id {:refresh false :body {:doc attributes}})]
         (log/debugf "detect-swarm - parent: %s - resource-id: %s - scripted-edit: %s" parent resource-id response)))))
+
+(defn complete-param
+  [{:keys [id acl] :as _deployment} deployment-parameter]
+  (-> deployment-parameter
+      (assoc :parent id
+             :created-by "internal"
+             :resource-type dep-param/resource-type
+             :acl acl)
+      (crud/new-identifier dep-param/resource-type)
```

Comment:
@0xbase12 I think this will not create the predictable UUID like here: 

https://github.com/nuvla/api-server/blob/issue_tasklist_3324/code/src/com/sixsq/nuvla/server/resources/deployment_parameter.clj#L29-L33

```clojute
(defn parameter->id
  [{:keys [parent node-id name] :as _parameter}]
  (->> (str/join ":" [parent node-id name])
       u/from-data-uuid
       (str resource-type "/")))
```

Reply:
Yes it will because crud/new-identifier is a defmethod https://github.com/nuvla/api-server/blob/5aef82e66a5b6fe41e4183b9f3bb25d63123fbfc/code/src/com/sixsq/nuvla/server/resources/deployment_parameter.clj#L61